src/interfaces/ITokenHopper.sol

```diff
@@ -45,7 +45,6 @@ interface ITokenHopper {
     struct HopperConfiguration {
         // Initial Funds
         address token;              // Each hopper will hold exactly one token type.
```

Comment:
make this an IERC20 to avoid conversion

Reply:
seems OK to me to convert in 2 places and reduce interface imports. we'd also have to convert _from_ IERC20 to address in one place if we made the change (at least with the interfaces as they are now)